Gravatar/GravatarService.cs

```diff
@@ -30,6 +31,7 @@ public class GravatarService : IAvatarService
         private readonly IImageCache _cache;
         private readonly IImageNameProvider _avatarImageNameProvider;
 
+        private const string GitHubPrivateEmailSuffix = "@users.noreply.github.com";
```

Comment:
RussKie standard comment: const is not version safe, use static readonly

Reply:
I would agree if it were publicly visible, but it's private, so it doesn't change anything... Anyway, I don't mind changing it if you prefer.

Reply:
correct, drop this